### DISCUSSION TITLE
Ísland tekur yfir heiminn - 2021/8/4

### DISCUSSION THREAD

**Discussion by giscus** [Biskupsreglan/biskupssidan#27] [vefsida-ummæli]
# Ísland tekur yfir heiminn - 2021/8/4

Vídeó notað fyrir stjórnmálaflokkin okkar

https://biskupsreglan.vercel.app/aukaefni/island-strid

**Comment by MatthiasGudni** [comment]
[Það styttist í það](https://www.youtube.com/watch?v=TDeGhkF5_Oo) 💪💪💪
